README.md

```diff
@@ -160,25 +160,25 @@ If you need additional flexibility, you can use the following building blocks to
 1. How far along the step is the user?
 1. Does the user need to be alerted about an upcoming maneuver?
 
-The library compares the user from the route and decides upon each one of these parameters and acts accordingly. The developer is told what is happening behind the scenes via `NSNotification`.
+The library compares the user from the route and decides upon each one of these parameters and acts accordingly. The developer is told what is happening behind the scenes via notifications.
```

Comment:
Notifications is a known concept for iOS devs so a common name here that works equally good for Swift and Obj-C shouldn't cause any confusion.